frontend/views/pages/PayGroup.vue

```diff
@@ -1,122 +1,128 @@
+      ) {month} contributions
+      i18n.has-text-1(
+        tag='p'
+        :args='{ amount: `${fakeStore.currency}${paymentStatus.amountTotal}` }'
+      ) {amount} in total
+
+      ul.c-payments
+        li.c-payments-item(
+          v-for='(user, index) in fakeStore.usersToPay'
           :key='user.name'
         )
-          th
-            avatar(
-              :src='user.avatar'
-              :alt='L("{username}\'s avatar", { username: user.name} )')
-
-            span.c-table-cell {{user.name}}
-
-          td
-            .c-status(v-if='statusIsToPay(user)')
-              i18n.button.is-small.c-table-cell(
-                tag='button'
-                @click='markAsPayed(user)'
-              ) Mark as paid
-
-              tooltip(
+          .c-info
+            avatar.c-avatar(:src='user.avatar')
+            div(role="alert")
+              i18n(
+                tag="p"
+                :args='{ total: `<b>${fakeStore.currency}${user.amount}</b>`, user: `<b>${user.name}</b>`}'
+              ) {total} to {user}
+              i18n.has-text-1(
+                v-if='statusIsPending(user)'
+                :args='{ name: getUserFirstName(user.name) }'
+              ) Waiting for {name} confirmation...
+              i18n.has-text-success(
+                v-if='statusIsCompleted(user)'
+              ) Payment confirmed!
+              i18n.has-text-weight-normal.has-text-warning(
                 v-if='statusIsRejected(user)'
-                direction='right-start'
-              )
-                i.icon-exclamation-triangle.has-text-warning.is-size-6.c-icon-badge
-
-                template(slot='tooltip')
-                  strong.has-text-weight-bold Payment Declined
-                  i18n.has-text-weight-normal(
-                    tag='p'
-                    :args="{ name: 'bob' }"
-                  ) {name} didn&rsquo;t confirm your payment. Please mark as payed only when it&rsquo;s done.
-
-            .c-status(v-else-if='statusIsPending(user)')
-              i.icon-paper-plane.c-status-icon
-              p
-                i18n.has-text-1(:args='{ name: getUserFirstName(user.name), admiration: getCustomAdmiration(index) }')
-                  | {admiration}! Waiting for {name} confirmation.&nbsp;
-                i18n.is-unstyled.link(tag='button' @click='cancelPayment(user)')
-                  | Cancel payment
-
-            .has-text-success.c-status(v-else-if='statusIsCompleted(user)')
-              i.icon-check-circle.is-size-5.c-status-icon
-              i18n Payment sent!
-
-          td
-            | {{fakeStore.currency}}{{user.amount}}
+                :args="{ name: getUserFirstName(user.name) }"
+              ) The payment was not received by {name}.
+
+          .buttons.is-start.c-ctas
+            router-link.button.is-small.is-outlined(
+              v-if='statusIsRejected(user)'
+              to='messages/liliabt'
+            ) {{ L('Send Message...') }}
```

Comment:
I think this can be `i18n(tag='router-link' ...` (double-check that it works, just in case if it's finicky like `template` tag too, which doesn't work with `i18n`), but doing it this way is OK too

Reply:
It doesn't work because of `to` prop.